crates/iceberg/src/spec/manifest_list.rs

```diff
@@ -1120,15 +1236,17 @@ mod test {
 
     #[tokio::test]
     async fn test_manifest_list_writer_v2() {
-        let expected_manifest_list = ManifestList {
+        let snapshot_id = 377075049360453639;
+        let seq_num = 1;
+        let mut expected_manifest_list = ManifestList {
             entries: vec![ManifestListEntry {
                 manifest_path: "s3a://icebergdata/demo/s1/t1/metadata/05ffe08b-810f-49b3-a8f4-e88fc99b254a-m0.avro".to_string(),
                 manifest_length: 6926,
                 partition_spec_id: 1,
                 content: ManifestContentType::Data,
-                sequence_number: 1,
-                min_sequence_number: 1,
-                added_snapshot_id: 377075049360453639,
+                sequence_number: UNASSIGNED_SEQUENCE_NUMBER,
```

Comment:
On the manifest list the snapshot should be assigned, maybe we can add a note here that it should be fixed once we get the snapshots generation in there.

Reply:
I'm sorry. I'm not sure what you mean.🤔 In here the `snapshot id` is assigned. What's the snapshots generation?